### ISSUE TITLE
PT2 Module Not Found error

### ISSUE THREAD

**Issue by ImagePlant** [jentron/Blender-PT2#15]
When this addon is installed and enabled in Blender, a Module Not Found error occurs.
![Annotation 2020-07-03 223806](https://user-images.githubusercontent.com/57046940/86505846-6ff3d080-bd7e-11ea-922f-04db898fe22e.jpg)

**Comment by jentron** [comment]
Please rename the folder from Blender-PT2 to just PT2 and it should work?

**Comment by ImagePlant** [comment]
Yup that looks to have fixed the issue.  I can now import props and characters.

![poserbaby](https://user-images.githubusercontent.com/57046940/86520251-0f54aa00-bdf7-11ea-96cc-e05b0150e5e6.jpg)

**Comment by jentron** [comment]
Awesome. I updated the readme with a note to make that change.

**Comment by ImagePlant** [comment]
Cool.  I threw a post up on Renderosity giving a heads up.  Hopefully some coders may read that and help you if you want.

**Comment by jentron** [comment]
Do you have a link to the post? I know how to fix it, I just haven't bothered yet because it doens't bother me :) I'd rather get shape keys (aka morph targets) working on the character importer. :)

**Comment by ImagePlant** [comment]
I posted it in an existing thread:

https://www.renderosity.com/mod/forumpro/?thread_id=2948728&page_number=2#msg4393865

**Comment by ImagePlant** [comment]
Jentron, 

I created a thread about this addon on BlenderArtists.org.  Just trying to help spread the word.   Link to post is [here](https://blenderartists.org/t/poser-import-export-addon-has-been-revived/1239826)

**Comment by jentron** [comment]
Hmm, you're not "the" Geep, are you?

**Comment by ImagePlant** [comment]
yes I am

**Comment by jentron** [comment]
You already have an honorable mention on issue #12, then.
